spec/datadog/appsec/contrib/rack/integration_test_spec.rb

```diff
@@ -204,20 +222,73 @@
           it_behaves_like 'a GET 200 span'
           it_behaves_like 'a trace with AppSec tags'
           it_behaves_like 'a trace with AppSec events'
+
+          context 'and a blocking rule' do
+            let(:appsec_ruleset) do
```

Comment:
This appears multiple times intentionally: I'm still figuring a way to better test using variable rulesets, and I don't want to commit to a messy design yet.

Reply:
I factored that out in a crude way that may be indicative of a future direction.